package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "hp-admin",
-  "version": "0.1.0",
+  "version": "0.2.1",
   "private": true,
   "scripts": {
     "start": "rm -rf ./node_modules/.cache/babel-loader && node scripts/start.js",
     "start:holofuel": "REACT_APP_HOLOFUEL_APP=true npm start",
     "build": "node scripts/build.js",
     "build:holofuel": "REACT_APP_HOLOFUEL_APP=true node scripts/build.js",
+    "hc:start-manual": "holochain -c conductor-config-manuals.toml",
```

Comment:
`conductor-config-manuals.toml` should be `conductor-config-manual.toml`

Reply:
Good catch! :)